File: Easy-exercises/Gotham City/Gotham_City.cpp
#include<iostream>
using namespace std;
int main(){
int criminals;
    cout <<"Enter number of criminals: " << endl;
    cin >> criminals;
    criminals > 10 ? cout <<"Good Luck out there! " << endl
    : (criminals >= 5 && criminals < 10) ? cout <<"Help me Batman " << endl 
    : cout <<"I got this! " <<endl;
    return 0;
}

File: Easy-exercises/Gotham City/Gotham_City.js
const gothamCity = (criminals) => {
  criminals > 10
    ? console.log("Good Luck out there!")
    : criminals >= 5 && criminals < 10
    ? console.log("Help me Batman")
    : console.log("I got this!");
};
gothamCity(parseInt(prompt("Enter number of criminals")));
